server/chat-commands/core.ts

```diff
@@ -1275,15 +1308,15 @@ exports.commands = {
 			return;
 		}
 		if (!target) return this.errorReply("Provide a valid format.");
-		let originalFormat = Dex.getFormat(target);
+		const originalFormat = Dex.getFormat(target);
 		// Note: The default here of [Gen 7] Pokebank Anything Goes isn't normally hit; since the web client will send a default format
-		let format = originalFormat.effectType === 'Format' ? originalFormat : Dex.getFormat('[Gen 7] Pokebank Anything Goes');
-		if (format.effectType !== 'Format') return this.popupReply("Please provide a valid format.");
+		const tier = originalFormat.effectType === 'Format' ? originalFormat : Dex.getFormat('[Gen 7] Pokebank Anything Goes');
+		if (tier.effectType !== 'Format') return this.popupReply("Please provide a valid format.");
```

Comment:
Why did you rename `format` to `tier`?

Inside PS, `tier` always refers to a banlist, while `format` refers to a ruleset for a playable game.

Reply:
(This should be renamed back.)

Reply:
Like, UUBL is a tier, Random Battle is a format.